plugins/rapid7_insightappsec/bin/komand_rapid7_insightappsec

```diff
@@ -6,7 +6,7 @@ from sys import argv
 
 Name = "Rapid7 InsightAppSec"
 Vendor = "rapid7"
-Version = "1.0.3"
+Version = "2.0.0"
```

Comment:
```suggestion
Version = "1.0.4"
```

Reply:
Tested in staging. Changing the `name:` plugin spec field of the action without changing the plugin spec object name does not require a major version bump.